pkg/kubectl/rollout_status.go

```diff
@@ -105,7 +105,7 @@ func (s *DaemonSetStatusViewer) Status(namespace, name string, revision int64) (
 		return "", false, err
 	}
 	if daemon.Spec.UpdateStrategy.Type != extensions.RollingUpdateDaemonSetStrategyType {
```

Comment:
There is a second DaemonSet strategy in the works (https://github.com/kubernetes/community/pull/977) that should provide status. In reality, we should never have a strategy that does not provide status so this check should really be like the StatefulSet check:
```go
if daemon.Spec.UpdateStrategy.Type == extensions.OnDeleteDaemonSetStrategyType {
```